.circleci/config.yml

```diff
@@ -0,0 +1,160 @@
+defaults: &defaults
+  docker:
+    - image: circleci/node:10-browsers
+  working_directory: ~/origin
+
+db: &db
+  docker:
+    - image: circleci/node:10-browsers
+    - image: circleci/postgres:9.6-alpine-ram
+      environment:
+        POSTGRES_USER: postgres
+        POSTGRES_DB: travis_ci_test
+  working_directory: ~/origin
+
+version: 2
+jobs:
+  Setup:
+    <<: *defaults
+    steps:
+      - run:
+          name: Clone Repo
+          command: |
+            git clone --depth=1 -b $CIRCLE_BRANCH https://github.com/OriginProtocol/origin.git ~/origin
+      - restore_cache:
+          key: yarn-packages-v3-{{ checksum "yarn.lock" }}
+      - run:
+          name: Yarn Install
+          command: |
+            yarn install --frozen-lockfile
+            curl -sfL https://install.goreleaser.com/github.com/tj/node-prune.sh | sh
+            ./bin/node-prune
+      - save_cache:
+          key: yarn-packages-v3-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.cache/yarn
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  Lint:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - run: yarn lint
+
+  Marketplace:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - run: yarn run test --prefix dapps/marketplace
+
+  GraphQL:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - run: yarn run test --prefix packages/graphql
+
+  EventCache:
+    <<: *db
+    environment:
+      DATABASE_URL: postgres://postgres@localhost/travis_ci_test
+    steps:
+      - attach_workspace:
+          at: .
+      - run: ./node_modules/.bin/lerna run migrate --scope @origin/event-cache
+      - run: yarn run test --prefix packages/event-cache
+
+  Bridge:
+    <<: *db
+    environment:
+      DATABASE_URL: postgres://postgres@localhost/travis_ci_test
+    steps:
+      - attach_workspace:
+          at: .
+      - run: ./node_modules/.bin/lerna run migrate --scope @origin/bridge
+      - run: yarn run test --prefix infra/bridge
+
+  Contracts:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: .
+      - run: yarn run test --prefix packages/contracts
+
+  Discovery:
+    <<: *db
+    environment:
+      DATABASE_URL: postgres://postgres@localhost/travis_ci_test
+      AFFILIATE_ACCOUNT: 0x821aea9a577a9b44299b9c15c88cf3087f3b5544
```

Comment:
These ACCOUNT definitions are from origin-js. Not needed anymore.

Reply:
Nice, have removed these now